src/hotspot/share/opto/macro.cpp

```diff
@@ -1078,7 +1023,7 @@ bool PhaseMacroExpand::eliminate_allocate_node(AllocateNode *alloc) {
     return false;
   }
 
-  extract_call_projections(alloc);
+  alloc->extract_projections(&_callprojs, false/*separate_io_proj*/, false/*do_asserts*/);
```

Comment:
Please add a whitespace between the argument and the comment: `false /*separate_io_proj*/`. There are multiple occurrences of this.

Reply:
updated them all.